components/authz-service/engine/opa/opa.go

```diff
@@ -360,6 +362,7 @@ func (s *State) FilterAuthorizedPairs(
 		"pairs":    pairs,
 	}
 
+	// NB: V1 only, so s.store used here
```

Comment:
Added an explicit line like this on each `evalQuery` to show that it has been consciously considered.